lib/evss/base_service.rb

```diff
@@ -51,7 +52,7 @@ def conn
         faraday.use      :breakers
         faraday.use      EVSS::ErrorMiddleware
         faraday.use      Faraday::Response::RaiseError
-        faraday.response :json, content_type: /\bjson$/
+        faraday.use      EVSS::SnakeCaseMiddleware, content_type: /\bjson$/
```

Comment:
Can this just use: 

`conn.response :snakecase`

from: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/client/middleware/response/snakecase.rb